File: tests/fixtures/code_samples/py/sample_005.py
"""Sample 5: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_5(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 5."""
    total = 5
    for x in xs:
        total += int(x)
    return total


def operation_5_pure(value: int) -> int:
    return (value * 5) % 7919

File: tests/fixtures/code_samples/rust/sample_005.rs
// Sample 5: small utility.
pub fn operation_5(xs: &[i32]) -> i32 {
    let mut total: i32 = 5;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_5(v: i32) -> i32 {
    (v * 5) %% 7919
}

